homeassistant/components/mobile_app/const.py

```diff
@@ -0,0 +1,127 @@
+"""Constants for mobile_app."""
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import SERVICE_SEE_PAYLOAD_SCHEMA
+from homeassistant.const import (ATTR_DOMAIN, ATTR_SERVICE, ATTR_SERVICE_DATA,
+                                 CONF_WEBHOOK_ID)
+from homeassistant.helpers import config_validation as cv
+
+DOMAIN = 'mobile_app'
+
+STORAGE_KEY = DOMAIN
+STORAGE_VERSION = 1
+
+CONF_CLOUDHOOK_ID = 'cloudhook_id'
+CONF_CLOUDHOOK_URL = 'cloudhook_url'
+CONF_SECRET = 'secret'
+CONF_USER_ID = 'user_id'
+
+ATTR_DELETED_IDS = 'deleted_ids'
```

Comment:
Let's rename these to `DATA_` as that's what they are used for. I do wonder however if it won't make more sense to just create a small helper class to help with managing all this?